client/packages/coldchain/src/Equipment/DetailView/Tabs/Details.tsx

```diff
@@ -147,7 +146,7 @@ export const Details = ({ draft, onChange }: DetailsProps) => {
                             },
                           })
                         }
-                        isCatalogue={isCatalogue}
+                        disabled={isCatalogue}
```

Comment:
Moved PropertyInput to be a common component, so made `disabled` a bit more generic